dags/maxtext_pathways/configs/utils.py

```
@@ -109,6 +122,63 @@ def generate_derived_parameters(dag_params: dict, dag_id: str) -> dict:
+        command=exec_command,
+        stderr=True,
+        stdin=False,
+        stdout=True,
+        tty=False,
+    )
+    logging.info(
+        "Successfully executed kill command on %s. Response: %s",
+        target_pod_name,
+        resp,
+    )
+    return True
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)